src/components/download/DownloadDialog.js

```diff
@@ -1,97 +1,196 @@
 import i18n from '@dhis2/d2-i18n'
-import {
-    Modal,
-    ModalTitle,
-    ModalContent,
-    ModalActions,
-    Button,
-    ButtonStrip,
-} from '@dhis2/ui'
-import React, { useState, useMemo } from 'react'
+import { Button, ButtonStrip } from '@dhis2/ui'
```

Comment:
To me it makes more sense that this component is called "DownloadSettingsPanel"

Reply:
Renamed to `DownloadSettings`